pettingzoo/test/example_envs/generated_agents_parallel_cust_agentid_v0.py

```diff
@@ -0,0 +1,164 @@
+from typing import Union
+
+import gymnasium
+import numpy as np
+from gymnasium.utils import seeding
+
+from pettingzoo import ParallelEnv
+from pettingzoo.utils import conversions, wrappers
+
+
+def env(**kwargs):
+    env = raw_env(**kwargs)
+    env = wrappers.AssertOutOfBoundsWrapper(env)
+    env = wrappers.OrderEnforcingWrapper(env)
+    return env
+
+
+def raw_env(**kwargs):
+    return conversions.parallel_to_aec(parallel_env(**kwargs))
+
+
+def get_type(agent: tuple[str, int]):
+    return agent[0]
+
+
+class parallel_env(ParallelEnv[tuple[str, int], np.ndarray, Union[int, None]]):
```

Comment:
My only thought is that this is a bit confusing in that when you look at it, it doesn't immediately become clear that the first arg is the AgentID, and that the others are observation, etc.

Reply:
I'm not currently aware of any way to provide named generic arguments, but I agree that many generic arguments is kind of confusing. We only have 3 so it's not too bad yet.